config.json

```diff
@@ -1,6 +1,8 @@
 {
   "language": "Haskell",
   "active": true,
+  "solution_pattern": "example.*[.]hs",
+  "ignore_pattern": "[Ee]xample",
```

Comment:
because of https://github.com/exercism/trackler/blob/58b942a857b61de29c9d799e0c61ec57bbfa564a/lib/trackler/implementation.rb#L71, the `[Ee]` is unnecessary. I think it should just be `example`.